src/components/content-types/workspace/ChangedDocsList.css

```diff
@@ -0,0 +1,4 @@
+.UnseenChangesDoc-emptyState {
```

Comment:
more for-later feedback but i think we should make an effort to eliminate all styling outside of the presentational components. it's just so much easier to build UIs when you don't have to make these kinds of decisions per-component and when you don't have to track down weird behaviour per-component